src/core/Akka.Remote/Serialization/PrimitiveSerializers.cs

```diff
@@ -29,12 +30,18 @@ public sealed class PrimitiveSerializers : SerializerWithStringManifest
         internal const string Int32ManifestNetFx = "System.Int32, mscorlib";
         internal const string Int64ManifestNetFx = "System.Int64, mscorlib";
 
+        private readonly bool _useNeutralPrimitives;
+
         /// <summary>
         /// Initializes a new instance of the <see cref="PrimitiveSerializers" /> class.
         /// </summary>
         /// <param name="system">The actor system to associate with this serializer. </param>
-        public PrimitiveSerializers(ExtendedActorSystem system) : base(system)
+        public PrimitiveSerializers(ExtendedActorSystem system, Config config) : base(system)
```

Comment:
Do we need the config parameter here or can that be extracted from `System.Settings`?

Reply:
Have to do this here because of our own convention.
Serializer settings are injected into the serializer from `akka.actor.serialization-settings`, and if the configuration for specific serializer is defined in this configuration setting, the serializer is then instantiated with the configuration injected in the constructor.
The code will fail if that config parameter is ommited.

Reply:
Do you prefer to have this setting separate from our convention on providing settings to serializers?

Reply:
This is fine then